tests/EntryPoints/SearchEntitiesApiTest.php

```diff
@@ -19,7 +19,7 @@
 use Wikibase\DataModel\Statement\StatementList;
 
 /**
- * @covers \ProfessionalWiki\WikibaseExport\EntryPoints\SearchEntitiesApi
+ * @see \ProfessionalWiki\WikibaseExport\EntryPoints\SearchEntitiesApi
```

Comment:
Is this change because `@covers` limits the coverage report to only the explicitly referenced class?

Reply:
Yes